lib/pbench/test/unit/server/test_cache_manager.py

```diff
@@ -406,8 +406,8 @@ def mock_run(args, **_kwargs):
         "errmsg,expected",
         (
             ("Error unpacking tarball\n", "Error unpacking tarball"),
-            ("This is a message we'll find too long", "This is a message we'll f"),
-            ("One line\nTwo line\nThree line\nFour line\n", "One line\nTwo line"),
+            ("This is a message we'll find too long", "[TRUNC]This is a message we'll f"),
+            ("One line\nTwo line\nThree line\nFour line\n", "[TRUNC]One line\nTwo line"),
```

Comment:
Apparently Black is unhappy with this.  🦐

Reply:
Apparently, after re-applying black multiple times while hacking around during presentations, I neglected to blacken the code one final time before pushing the commit. Figures.

Reply:
Multitasking is hard.